galley/pkg/config/analysis/msg/messages.yaml

```diff
@@ -535,3 +535,12 @@ messages:
     description: "user namespace should be injectable if Istio is installed with enableNamespacesByDefault enabled and neither injection label is set."
     template: "is enabled for Istio injection, as Istio is installed with enableNamespacesByDefault as true."
     url: "https://istio.io/latest/docs/reference/config/analysis/ist0148/"
+
+  - name: "ExternalNameServiceTypeInvalidPortName"
+    code: IST0150
+    level: Error
+    description: "Port name is not under naming convention. Proxy blocks communication to external services if service is of type ExternalName."
+    template: "Port name %s (port: %d, targetPort: %s) doesn't follow the naming convention of Istio port. Proxy will block communication to defined external service"
```

Comment:
I don't think this is accurate. It does not necessarily block traffic to external service, but rather incorrectly route unmatched traffic. Probably something like "Proxy could potentially prevent TCP traffic on the same port from being forwarded correctly" would be better.

Reply:
Yes, I can make that change. I just wonder will it "potentially prevent TCP traffic"? I cannot think of a case where it will actually succeed

Reply:
+1 to what @bianpengyuan said.